src/pilot/config_idrac.py

```diff
@@ -541,6 +541,22 @@ def reset_idrac(drac_client, ip_service_tag):
     drac_client.reset_idrac(wait=True)
 
 
+def get_lifecycle_out_of_recovery(drac_client, ip_service_tag):
+    LOG.info("Setting lifecycle settings on {}".format(ip_service_tag))
+    settings = {'Lifecycle Controller State': 'Enabled'}
+    drac_client.set_lifecycle_settings(settings)
+
+    LOG.info("Waiting for lifecycle controller to get out of"
+             "recovery mode")
+    job_id = drac_client.commit_pending_lifecycle_changes(reboot=False)
+    job_succeeded = wait_for_jobs_to_complete(
+        [job_id], drac_client, ip_service_tag)
+
+    if not job_succeeded:
+        raise RuntimeError("An error occurred while configuring "
```

Comment:
Would you mind changing the error to: "An error occurred while taking the iDRAC on xxx out of recovery mode"

Reply:
Sure Chris. I will change the error as suggested.
Thank you!